doc/tutorials/acrn_on_qemu.rst

```diff
@@ -186,10 +193,14 @@ Install ACRN Hypervisor
          insmod ext2
 
          echo 'Loading ACRN hypervisor ...'
-         multiboot --quirk-modules-after-kernel /boot/acrn.32.out
-         module /boot/bzImage Linux_bzImage
+         multiboot --quirk-modules-after-kernel /boot/acrn.32.out  root=/dev/vda5
+         module /boot/vmlinuz-5.10.109 Linux_bzImage
       }
 
+     .. note::
+     If your rootfs partition isn't vda5, please change it to match with yours.
+     vmlinuz-5.10.109 is the kernel image of Service VM. 
```

Comment:
We have this very short line that instructs the user to build the Service VM kernel just like it's described in the GSG. The GSG creates a Debian package that's transfered from the dev machine over to the target system. Is this out recommendation here?